src/main/java/org/jabref/logic/importer/fileformat/CffImporter.java

```diff
@@ -92,73 +103,100 @@ public CffIdentifier() {
+        if (preferredCitation != null) {
+            if (preferredCitation.has("type")) {
+                String typeValue = preferredCitation.get("type").asText();
+                StandardEntryType entryType = mapType(typeValue);
+                entry.setType(entryType);
+            }
+            if (preferredCitation.has("title")) {
+                entryMap.put(StandardField.TITLE, preferredCitation.get("title").asText());
+            }
+            if (preferredCitation.has("doi")) {
+                entryMap.put(StandardField.DOI, preferredCitation.get("doi").asText());
+            }
+            if (preferredCitation.has("authors")) {
+                List<String> authorsList = new ArrayList<>();
+                preferredCitation.get("authors").forEach(authorNode -> {
+                    String givenName = authorNode.has("given-names") ? authorNode.get("given-names").asText() : "";
+                    String familyName = authorNode.has("family-names") ? authorNode.get("family-names").asText() : "";
+                    authorsList.add((givenName + " " + familyName).trim());
+                });
+                String authors = String.join(" and ", authorsList);
+                entryMap.put(StandardField.AUTHOR, authors);
+            }
+            if (preferredCitation.has("journal")) {
+                entryMap.put(StandardField.JOURNAL, preferredCitation.get("journal").asText());
+            }
+            if (preferredCitation.has("volume")) {
+                entryMap.put(StandardField.VOLUME, preferredCitation.get("volume").asText());
+            }
+            if (preferredCitation.has("issue")) {
+                entryMap.put(StandardField.ISSUE, preferredCitation.get("issue").asText());
+            }
+            if (preferredCitation.has("year")) {
+                entryMap.put(StandardField.YEAR, preferredCitation.get("year").asText());
+            }
+            if (preferredCitation.has("start") && preferredCitation.has("end")) {
+                String pages = preferredCitation.get("start").asText() + "-" + preferredCitation.get("end").asText();
+                entryMap.put(StandardField.PAGES, pages);
+            }
+        }
+    }
+
+    private void mainCffContentMethod(CffFormat citation, Map<Field, String> entryMap, BibEntry entry) {
+        if (!entryMap.containsKey(StandardField.TITLE) && citation.values.containsKey("title")) {
+            entryMap.put(StandardField.TITLE, citation.values.get("title"));
+        }
+        if (!entryMap.containsKey(StandardField.AUTHOR) && citation.authors != null && !citation.authors.isEmpty()) {
+            List<String> authorsList = new ArrayList<>();
+            for (CffAuthor author : citation.authors) {
+                String givenName = author.values.getOrDefault("given-names", "");
+                String familyName = author.values.getOrDefault("family-names", "");
+                authorsList.add((givenName + " " + familyName).trim());
+            }
```

Comment:
Seems to be code duplication compared to above. But also slightly different. Maybe a code comment should be added why the code at `preferredCitationMethode` is different from here.